utils/documentation_tests.txt

```diff
@@ -49,3 +49,17 @@ src/transformers/models/wav2vec2/modeling_wav2vec2.py
 src/transformers/models/wav2vec2/tokenization_wav2vec2.py
 src/transformers/models/wav2vec2_with_lm/processing_wav2vec2_with_lm.py
 src/transformers/models/wavlm/modeling_wavlm.py
+src/transformers/models/bart/modeling_bart.py
+src/transformers/models/mbart/modeling_mbart.py
+src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py
+src/transformers/models/marian/modeling_marian.py
+src/transformers/models/pegasus/modeling_pegasus.py
+src/transformers/models/blenderbot/modeling_blenderbot.py
+src/transformers/models/blenderbot_small/modeling_blenderbot_small.py
+src/transformers/models/plbart/modeling_plbart.py
+src/transformers/generation_utils.py
+docs/source/quicktour.mdx
+docs/source/task_summary.mdx
+src/transformers/models/resnet/modeling_resnet.py
+src/transformers/models/glpn/modeling_glpn.py
+src/transformers/models/canine/modeling_canine.py
```

Comment:
Hi, @NielsRogge 

I know this PR is still in draft. But do you happen to know why there are these many lines added which are mostly duplicated.

This also happened in your previous PR (#16199).